src/main/groovy/org/gradle/android/workarounds/JdkImageWorkaround.groovy

```diff
@@ -53,16 +53,28 @@ class JdkImageWorkaround implements Workaround {
         applyRuntimeClasspathNormalization(project)
 
         applyToAllAndroidVariants(project) { variant ->
-            variant.javaCompileProvider.configure { JavaCompile task ->
-                def jdkImageInput = getJdkImageInput(task)
-                if (jdkImageInput != null) {
-                    setupExtractedJdkImageInputTransform(project, getJvmHome(task))
-                    replaceCommandLineProvider(task, jdkImageInput)
+            if (Versions.CURRENT_ANDROID_VERSION <= VersionNumber.parse("7.4.0-alpha01")) {
+                variant.javaCompileProvider.configure { JavaCompile task ->
+                    jdkTransform(project, task)
+                }
+            } else {
+                project.afterEvaluate {
```

Comment:
why do we need `afterEvaluate` here?

Reply:
here correct me if I'm wrong, but my approach was based in:
because the `variant.javaCompilerProvider` is unavailable, I need to check if a compiler argument is defined for jdkImage. When I'm configuring the project, the AGP didn't create the compiler argument yet. That's the main reason. Ideas/suggestions?

Reply:
See above thread ^